layouts/wrapper.html

```diff
@@ -33,12 +33,20 @@
 		{{/outputBlock}}
 	</head>
 	<body class="o-hoverable-on">
+
+		{{#if showSkipLinks}}
+			<a class="u-visually-hidden" href="#primary-nav">Skip to navigation</a>
+			<a class="u-visually-hidden" href="#content">Skip to content</a>
```

Comment:
Nice! My only concern here is what if a new app forgets to add this. I.e. skip-link functionality with a broken anchor potentially worse to the user than not having one. It also makes it very hard to test.